lib/cocoapods/installer.rb

```diff
@@ -229,6 +238,7 @@ def generate_pods_project(generator = create_generator(installation_options.gene
         end
 
         write_lockfiles
+        SandboxDirCleaner.new(sandbox, pod_targets, aggregate_targets).clean!
```

Comment:
this does not seem to belong in the method `generate_pods_project`, why not within `clean_sandbox`?